README.md

```diff
@@ -37,3 +38,48 @@ Command | Usage
 /help | Brings up a quick menu to remind you of commands
 
 ## Setup
+1. Download the zip file from the green "Code" button above
+
+2. Extract the zip file into a folder
+3. Create a ```.txt``` file, but change its name to just ```.env```! 
+    3. **Note that there is NO name in front of  ```.env```**
+4. Open up the newly created ```.env``` file using Notepad or another of your choice
+5. Follow the ```Creating a Bot Account``` on this site:
+```https://discordpy.readthedocs.io/en/latest/discord.html```
+6. Copy the token from step 7 (of the site)
+7. Within ```.env```, you will have to add in **THREE** lines:
+    7. ```DISCORD_TOKEN = <the_token_you_obtained_from_the_prior_sites>```
+    7. ```DELETE_THRESHOLD = <integer_greater_than_0>``` 
+        7. More information in the Questions section
+    7. ```DB_URI = <database_URI>``` 
+        7. Ignore this if you aren't sure what this is. More information in the Questions section
+8. Go back to ```https://discordpy.readthedocs.io/en/latest/discord.html``` and complete the 
+```Inviting Your Bot``` section
+9. Run ```bot.py``` file and your bot should turn Online on Discord
+10. Message the bot whenever you feel like it!
+
+### Questions
+- How do I send out a message to vent my problems or request for advice?
+    - Simply message the bot like you were messaging anyone else on Discord! It will reply with a 
+    green check if it is properly received!
+    
+- How do I reply to other people?
+    - Use the command ```/read``` to view others' messages. You can use Discord's own reply feature
+    by right clicking the message then using ```Reply``` in order to send back a response!
+    
+- How do I view responses people sent back to my messages?
+    - Use the command ```/responses``` to see your responses on letters!
+    
+- What is ```DISCORD_TOKEN``` used for?
+    - It specifically links ```bot.py``` with the Discord bot so it knows what commands to use.
+    
+- What is ```DELETE_THRESHOLD``` used for?
```

Comment:
Maybe it would be better to put env vars in a table like what you did for commands